docs/source/sysadmin_guide/creating_genesis_block.rst

```diff
@@ -0,0 +1,131 @@
+**************************
+Creating the Genesis Block
+**************************
+
+The first node in a new Sawtooth network must create the genesis block to
+initialize the Settings transaction processor and specify the consensus
+mechanism. The settings in the genesis block enable other nodes to join the
+network and use these on-chain settings.
+
+Before you start the first Sawtooth node, use this procedure to create and
+submit the genesis block.
+
+.. important::
+
+   Use this procedure **only** for the first validator node on a Sawtooth
+   network. Skip this procedure if you are creating a node that will join an
+   existing Sawtooth network.
+
+1. Ensure that the required user and validator keys exist:
+
+   .. code-block:: console
+
+      $ ls ~/.sawtooth/keys/
+      {yourname}.priv    {yourname}.pub
+
+      $ ls /etc/sawtooth/keys/
+      validator.priv   validator.pub
+
+   If these key files do not exist, create them as described in
+   :doc:`generating_keys`.
+
+#. Become the ``sawtooth`` user. In the following commands, the prompt
+   ``[sawtooth@system]`` shows the commands that must be executed as
+   ``sawtooth``.
+
+   .. code-block:: console
+
+      $ sudo -u sawtooth -s
+      [sawtooth@system]$ cd /tmp
+
+#. Create a batch to initialize the Settings transaction family in the genesis
+   block.
+
+   .. code-block:: console
+
+      [sawtooth@system]$ sawset genesis --key /etc/sawtooth/keys/validator.priv -o config-genesis.batch
+
+#. Create and submit a proposal to initialize the PoET consensus settings. This
+   command sets the consensus algorithm to PoET simulator, and then applies the
+   required settings.
+
+   .. code-block:: console
+
+      [sawtooth@system]$ sawset proposal create --key /etc/sawtooth/keys/validator.priv \
+      -o config.batch \
+      sawtooth.consensus.algorithm=poet \
+      sawtooth.poet.report_public_key_pem="$(cat /etc/sawtooth/simulator_rk_pub.pem)" \
+      sawtooth.poet.valid_enclave_measurements=$(poet enclave measurement) \
+      sawtooth.poet.valid_enclave_basenames=$(poet enclave basename)
+
+   This is a complicated command. Here's an explanation of the options:
+
+   ``--key /etc/sawtooth/keys/validator.priv``
+         Signs the proposal with this node's validator key. By default, only the
+         ``sawtooth`` user on this node can change on-chain settings. A later
```

Comment:
"By default, only the ``sawtooth`` user on this node can change on-chain settings."

I do not think this is correct. Permissions to change settings are controlled by whoever owns the key. It would be more accurate to say "Only the key passed to `sawset genesis` may be used to change on-chain settings".